bootstrap

```diff
@@ -43,20 +55,21 @@ main(_) ->
 
     {ok, State} = rebar3:run(["compile"]),
     reset_env(),
-    os:putenv("REBAR_PROFILE", ""),
```

Comment:
By removing this you ensure that the clean command we're gonna run on lines 51/62 cleans the code you just built. This does not seem reasonable?